contracts/LifCrowdsale.sol

```diff
@@ -11,6 +11,10 @@ contract LifCrowdsale is Ownable, Pausable {
   // The token being sold
   LifToken public token;
 
+  // start and end of the public presale
+  uint256 public startPublicPresaleBlock;
+  uint256 public endPublicPresaleBlock;
```

Comment:
@nberger for what other variables? for private presale we dont need to store block range, the private presale can happen only between contract creation and before `publicPresaleStartBlock`